contracts/linear/src/epoch_actions.rs

```diff
@@ -14,10 +14,14 @@ const MAX_SYNC_BALANCE_DIFF: Balance = 100;
 /// during each epoch.
 #[near_bindgen]
 impl LiquidStakingContract {
-    pub fn epoch_stake(&mut self) -> bool {
+    pub fn epoch_stake(&mut self) -> PromiseOrValue<bool /* can continue */> {
```

Comment:
`can continue` is a bit hard to understand I think. I suggest we add more details about what the return value means and why it's designed like this.